src/components/Autocomplete.css

```diff
@@ -10,6 +10,8 @@
 
 .autocomplete .results-summary {
   margin: 10px;
+  direction: ltr;
```

Comment:
got it. even took a picture of the debugger with the page on rtl and replaced the not-found-msg with a hebrew one :)

<img width="681" alt="screen shot 2016-12-28 at 8 33 20 pm" src="https://cloud.githubusercontent.com/assets/6891569/21528745/67f5f320-cd3d-11e6-9395-235cfc54b727.png">